### ISSUE TITLE
<q> not supported, and can't insert quotes content using CSS

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/wai-tutorials#290]
`<q>` is not supported in any assistive technology. The fallback, quotes in the example do not read to VoiceOver, NVDA, or JAWS, they cannot be selected with highlight. Q appears to report to a11y API as `Role:editable text` which doesn't identify.

Inserting content 
`q:before (content: "";)`
does not report to the DOM. Let's not advise to use CSS content insertion. In fact, let's recommend it *NOT* be used. Probably need a WCAG failure.

Just have them insert them manually
`<q>"..."</q>`

**Comment by yatil** [comment]
I don’t know if that really matters as (1) most screen readers don’t read quotes in the text (although that can be configured) and (2) `<q>` elements are usually used in conjunction with direct speech. The example is also readable without quotes:

```
Helen Keller said, Self-pity is our worst enemy and if we yield to it, we can never do anything good in the world.
```
I also can select the quotes in Firefox (as I have noticed copying the sentence over from the tutorial), in Voiceover I can see the quotes in the black overlay but they don’t get read. That seems to be an issue with the verbosity settings… I will continue to investigate this.

**Comment by DavidMacDonald** [comment]
Testing further... it's an IE problem.
http://davidmacd.com/blog/inserting-content-with-css-accessibility.html

Remove show stopper status.

**Comment by WobleDuck** [comment]
Yes Eric, you have this correctly.  Also, I'm sorry, I just couldn't
control myself on the 200% an 16px.

The point size on a 16px letter with a 13in monitor and 1600x900 resolution
(pretty normal) is computed as follows: pt/px= (13*cos(30)*72)/1600 so 16px
=16px*pt/px = 8.1pt.  On a 23in monitor that is 14.3pt. 200% of that is
16.2px and 28.3px.  The former is readable with additional magnification
the latter is readable for me at 8in, but remember I'm at the top of the
scale 6/24 to 6/30 corrected depending on my health.

Sorry, Wayne
